src/otx/core/model/base.py

```diff
@@ -123,7 +124,7 @@ def __init__(
 
         # this line allows to access init params with 'self.hparams' attribute
         # also ensures init params will be stored in ckpt
-        self.save_hyperparameters(logger=False, ignore=["model", "optimizer", "scheduler", "metric"])
+        self.save_hyperparameters(logger=False, ignore=["optimizer", "scheduler", "metric"])
```

Comment:
It's just a question. Why is 'model' removed?

Reply:
It is a kind of imprint remaining from `OTXLitModule`. `OTXLitModule.__init__()` had `model` argument